src/components/login.jsx

```diff
@@ -1,19 +1,45 @@
 import React from "react";
 import { GoogleLogin } from "react-google-login";
+import Home from "./home";
 
 class Login extends React.Component {
   constructor (props) {
     super(props);
+    this.state = {
+      loading: false,
+      error: false,
+      success: false,
+      userID: ""
+    };
   }
 
   responseGoogle (response) {
-    //get email from response -> 
-    console.log(response.profileObj.email);
+    this.setState({ loading: true });
+    fetch("http://localhost:3060/api/v1/users", {
+      method: "POST",
+      body: JSON.stringify({
+        "email": response.profileObj.email
+      }).then(res => res.json())
+        .then(json => {
+          this.setState({ success: true, loading: false, userID: response.profileObj.googleId });
```

Comment:
This is incorrect - the user endpoint returns the author `id` from the database, so you should be storing THAT in state, not the `googleId` returned from the login component. Refer back to the comments in Slack where this was discussed yesterday.